reconcile/test/test_utils_smtp_client.py

```diff
@@ -0,0 +1,83 @@
+# from reconcile.utils.secret_reader import SecretReader
+
+
+from typing import Any
+
+import pytest
+
+from reconcile.utils.smtp_client import SmtpClient
+
+
```

Comment:
since you're not mocking `smtplib.SMTP` anywhere in this file I understand this is using `smtpdfix`. Would it be worth mentioning it?

Reply:
I've added a note in the test file.